app/portainer/services/modalService.js

```diff
@@ -225,6 +225,24 @@ angular.module('portainer.app').factory('ModalService', [
       );
     };
 
+    service.upgradeNotification = function () {
+      bootbox.dialog({
+        size: 'extra-large',
+        title: 'Upgrade notification',
+        message: `You are currently using version 1.x of Portainer, which is no longer supported maintained or enhanced. Continuing to use this version is at your own risk.
+        <br/><br/>Please upgrade immediately by using the tag <code>portainer/portainer-ce-2.0.1</code>
+        <br/><br/>If you decide to stay on version 1, please use the tag <code>portainer/portainer:1.24.2</code> to prevent automatically upgrading to version 2.`,
+        onEscape: false,
+        backdrop: false,
```

Comment:
why disable backdrop?

Reply:
We want user to read this notification and migrate to 2.x. Disable backdrop in bootbox will force user to click "close" button. In this way, some of user will read our notification and think about it.

Reply:
by docs http://bootboxjs.com/documentation.html#bb-option-backdrop if backdrop is null, the backdrop will be displayed and clicking on it won't have effect. if backdrop is false, it won't be visible, so the background app will be visible

Reply:
Thank you, I will change to null

Reply:
We don't use backdrop to hide any content. Just let background app display. "false" is enough.

Reply:
but false will hide the backdrop.
![image](https://user-images.githubusercontent.com/1381655/152794475-e5f73d0d-13e6-4391-8aff-7d0dbd48bb57.png)

Reply:
I see that clicking the backdrop doesn't do anything on any other modules. so IMO `onEscape: false` is enough